composer.json

```diff
@@ -17,19 +17,21 @@
         "phpstan/phpstan": "^1.10",
         "phpstan/phpstan-strict-rules": "^1.5",
         "phpunit/phpunit": "^10.4",
+        "symfony/console": "^6.3",
         "symplify/monorepo-builder": "^11.2",
         "symplify/phpstan-rules": "^12.4",
+        "t3docs/cli": "@dev",
```

Comment:
I am not sure that this is correct in the main composer.json as the other t3docs packages are not included here.